### PR TITLE
Add Maintain Locations

### PR DESCRIPTION
**Pull request by dbnorth**

### CODE REVIEW

File: src/components/admin/maintain/locations/LocationDialogBody.vue [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+        <v-row v-if="props.isEdit" class="pt-0 mt-0"> </v-row>
+        <v-row :class="props.isEdit ? '' : 'mt-2'">
+          <v-col>
+            <v-card-subtitle
+              class="pl-0 pb-2 font-weight-semi-bold text-darkBlue"
+            >
+              Room Name
+            </v-card-subtitle>
+            <v-text-field
+              placeholder="Trumpet"
```
Comment (AndrewClubb) [inline src/components/admin/maintain/locations/LocationDialogBody.vue:95]: These placeholders should be updated

File: src/components/admin/maintain/locations/MaintainLocations.vue [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+
+// Filtering
+
+const filterMenuBool = ref(false);
+
+const searchInput = ref("");
+
+// Search filter
+// Filters the list of locations by first and last name, based on searchInput
+function searchAndFilterList() {
```
Comment (AndrewClubb) [inline src/components/admin/maintain/locations/MaintainLocations.vue:37]: The search bar doesn't filter locations

File: src/components/admin/maintain/locations/MaintainLocationCard.vue [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+            {{ locationData.status }}
+          </v-chip>
+          <v-btn
+            flat
+            size="small"
+            class="font-weight-bold mt-0 ml-4 text-none text-blue bg-white flatChipBorder"
+            @click="createOrEditDialog = true"
+          >
+            Edit
+          </v-btn>
```
Comment (AndrewClubb) [inline src/components/admin/maintain/locations/MaintainLocationCard.vue:71]: We are moving the edit buttons to the bottom right. Would you please move this one

**Review by AndrewClubb — CHANGES_REQUESTED** [review]

File: src/components/admin/maintain/locations/MaintainLocationCard.vue (hunk 3 above)
Comment (dbnorth) [reply src/components/admin/maintain/locations/MaintainLocationCard.vue:71]: I am confused all of the Edit buttons are top right.

Comment (dbnorth) [reply src/components/admin/maintain/locations/MaintainLocationCard.vue:76]: done

**Review by AndrewClubb — APPROVED** [review]

**Pull request merged**